corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html

```diff
@@ -56,6 +58,8 @@
     </div>
     {% endif %}
 
+    {{ request|toggle_tag_info:"CASE_LIST_TILE" }}
+    {% if request|toggle_enabled:'CASE_LIST_TILE' %}
```

Comment:
how come this is going behind a toggle now? Seems like it's already behind the `CASE_LIST_CUSTOM_XML` toggle.